### PR TITLE
Create AdministratorLanguageCest.php

### PR DESCRIPTION
**Pull request by kshitijSharma2014**
I couldn't understand where to place these tests as they test the multi-lingual feature of Joomla. These tests are also integrated tests. I have used a function for which I have also done a PR here : https://github.com/joomla-projects/joomla-browser/pull/105

### CODE REVIEW

**Comment by kshitijSharma2014** [comment]
@javigomez Please review
Thanks :)

**Comment by kshitijSharma2014** [comment]
To Test the PR please install multilingual Joomla. In the InstallWeblinksCest.php 
replace $I->installJoomlaRemovingInstallationFolder(); with $I->installJoomlaMultilingualSite();

Thanks

**Comment by kshitijSharma2014** [comment]
@javigomez checkes are failing here because I have created a function in joomla browser for delete menu. After it get merged, these checks might pass.

**Comment by yvesh** [comment]
Can you reformat it in the correct code style? (Tabs instead of spaces etc.). Some more tips in the code.

http://joomla.github.io/coding-standards/

Thanks for your contribution :+1:

File: tests/acceptance/administrator/AdministratorLanguageCest.php [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+  {
+    $I->am('Administrator');
+    $I->wantToTest('menu creation in /administrator/');
+    $I->doAdministratorLogin();
+    $I->amGoingTo('Navigate to MenuItems page in /administrator/');
+    $I->amOnPage('administrator/index.php?option=com_menus&view=items');
+    $I->waitForText('Menus: Items', '30', ['css' => 'h1']);
+    $I->expectTo('see MenuItem page');
+    $I->checkForPhpNoticesOrWarnings();
+    $title = "kshitij en-GB3".rand(1, 100);
```
Comment (yvesh) [inline tests/acceptance/administrator/AdministratorLanguageCest.php]: Can you use faker (`$this->faker`, offers many nice features to generate titles and all other stuff) here and not include your name. [https://github.com/fzaninotto/Faker](https://github.com/fzaninotto/Faker)

File: tests/acceptance/administrator/AdministratorLanguageCest.php [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    $I->amGoingTo("Home page");
+    $I->amOnPage("index.php/");
+    $I->waitForText("Test Lang en-GB", '30', ['css' => 'a']);
+    $I->click(['xpath' => "//li[2]/a/img"]);
+    $I->see("Page d'accueil");
+    $I->click(['xpath' => "//li[1]/a/img"]);
+    $I->see("Home");
+    $I->deleteMenuItem("Test Lang en-GB");
+    $I->deleteMenuItem("Test Lang fr-FR");
+
```
Comment (yvesh) [inline tests/acceptance/administrator/AdministratorLanguageCest.php]: Empty line

File: tests/acceptance/administrator/AdministratorLanguageCest.php [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    $I->amGoingTo('Navigate to Categories page in /administrator/');
+    $I->amOnPage('administrator/index.php?option=com_categories&extension=com_weblinks');
+    $I->waitForText('Weblinks: Categories', '30', ['css' => 'h1']);
+    $I->expectTo('see categories page');
+    $I->checkForPhpNoticesOrWarnings();
+    $I->amGoingTo('try to save a category with a filled title english');
+    $I->click(['xpath' => "//button[@onclick=\"Joomla.submitbutton('category.add')\"]"]);
+    $I->waitForText('Weblinks: New Category', '30', ['css' => 'h1']);
+    $salt = rand(1, 100);
+    $I->fillField(['id' => 'jform_title'], 'webcat-en ' . $salt);
```
Comment (yvesh) [inline tests/acceptance/administrator/AdministratorLanguageCest.php]: Same goes here, please use faker or the title.

File: tests/acceptance/administrator/AdministratorLanguageCest.php (hunk 1 above)
Comment (kshitijSharma2014) [inline tests/acceptance/administrator/AdministratorLanguageCest.php]: @yvesh I did a really bad mistake :( . I will correct and let you know.

**Comment by kshitijSharma2014** [comment]
@yves please review it 
thanks :)

**Comment by yvesh** [comment]
@kshitijSharma2014 What's the status on this? Can you please give an update in the next days? Else i will close it for now. Thank you!

**Comment by kshitijSharma2014** [comment]
@yvesh I am closing it right now. I will re-do a new PR with these changes. :)

**Pull request closed without merging**